src/modules/ordersTable/pure/ReceiptModal/FilledField.tsx

```diff
@@ -11,15 +11,15 @@ interface Props {
 }
 
 export function FilledField({ order }: Props) {
-  const { filledPercentage, formattedPercentage, fullyFilled } = order
+  const { executionData } = order
```

Comment:
Can you de-struct `executionData` as well?

Reply:
👌